server/server.cc

```diff
@@ -0,0 +1,181 @@
+#include <iostream>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <sstream>
+#include "user/User.h"
+#include "utils/serverUtils.h"
+
+int main() {
+    smart_home::User &user = smart_home::User::getInstance();
```

Comment:
same advise, group your logic into multiple small functions, we should aim for 15-20 lines of code ideally.